### DISCUSSION TITLE
`_UnwrapAll` not working when pinia stores is from npm package

### DISCUSSION THREAD

**Discussion by szulcus** [vuejs/pinia#2595] [Help and Questions]
### Reproduction

I can't reproduce it from playground

### Steps to reproduce the bug

For context, I use [Nuxt Layers](https://nuxt.com/docs/getting-started/layers) and `@pinia/nuxt` module.

1. Definie auto imports in `nuxt.config.ts`:
```ts
export default defineNuxtConfig({
	// ...
	imports: {
		dirs: ['stores', 'node_modules/@frontend/nuxt-layer-base/stores'],
	},
});
```
2. Use autoimported store from npm package:
```ts
const f24BaseStore = useF24BaseStore();
```
3. Use variable from this store:
```ts
console.log(f24BaseStore.toasts);
```
4. Show type definitions i get type of reactive variable (wrapped with `{ value: ...; [RefSymbol]: true; }`)

### Expected behavior

![image](https://github.com/vuejs/pinia/assets/48328739/2eeb9eb7-49b5-49d7-b468-be8e6c3ca030)

### Actual behavior

![image](https://github.com/vuejs/pinia/assets/48328739/a184a8f0-818e-4777-82a5-e57a40635614)

### Additional information

In local stores all works fine. For woraround I have this in `index.d.ts`:
```ts
import { UnwrapRef } from 'vue';
// import { _UnwrapAll } from 'pinia';
export {};

type _UnwrapAll<SS> = {
	[K in keyof SS]: UnwrapRef<SS[K]>;
};
declare global {
	const useF24BaseStore: () => _UnwrapAll<ReturnType<(typeof import('@frontend/nuxt-layer-base/stores/base'))['useF24BaseStore']>>;
}

```

**Comment by posva** [comment]
You need to ensure they all import from the same Vue package. Make sure pinia and vue are externalized
